platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/aibot/ChatWindowUi.kt

```diff
@@ -70,15 +70,23 @@ fun ChatWindowUi(state: ChatScreen.State, modifier: Modifier = Modifier) {
         }
       }
     }
+    if (state.isLoading) {
+      //       insert loading animation
```

Comment:
It looked like you had this done in your demo already, want to just add it in this PR?

Reply:
750f1ac0a65e1999a490405b671f03207cc00230